dev/docker-compose-sdp-anchor.yml

```diff
@@ -116,7 +116,7 @@ services:
       METRICS_ENABLED: "false"  # Metrics would be available at port 8082
       METRICS_EXTRAS_ENABLED: "false"
       SEP10_ENABLED: "true"
-      SEP10_HOME_DOMAINS: "localhost:8000, *.stellar.local" # Comma separated list of home domains
+      SEP10_HOME_DOMAINS: "localhost:8000, redcorp.stellar.local:8000, bluecorp.stellar.local:8000" # Comma separated list of home domains
```

Comment:
Wildcard domains doesn't work for now. Opened a ticket for anchor team : https://stellarorg.atlassian.net/browse/ANCHOR-612